kernel/README.md

```diff
@@ -0,0 +1,43 @@
+# kernel
```

Comment:
I suggest that the parent README.md file should incorporate a link to this README.md file for better cross-referencing and ease of navigation.

Reply:
Good point. Probably, should point out the current state of affairs. The top level readme also does not point out the recently added linuxkit infrastructure. I will have a look.

Reply:
Added another commit to add some info on the top-level README.md

Let me know what you think.